projects/ngx-avatar/src/lib/avatar.component.ts

```diff
@@ -253,20 +267,22 @@ export class AvatarComponent implements OnChanges, OnDestroy {
    * memberof AvatarComponent
    */
   private fetchAndProcessAsyncAvatar(source: AsyncSource): void {
+    if (this.avatarService.sourceHasFailedBefore(source)) {
```

Comment:
Any changes the source could work this time? Maybe due a temporary malfunction from the source's server. How could we deal with that?